packages/test/test-utils/src/TestSummaryUtils.ts

```diff
@@ -96,13 +100,15 @@ export async function createSummarizer(
     container: IContainer,
     summaryVersion?: string,
     gcOptions?: IGCRuntimeOptions,
+    registry?: ChannelFactoryRegistry,
```

Comment:
This function is not called in the tests. So, this update is not required.

Reply:
yes, but whenever we need to create a summarizer from a DataObject that requires a factory, it will fail without it.